### PR TITLE
Update readmes to match common design (Search category) 1/1

### PR DESCRIPTION
**Pull request by JonLavi**

### CODE REVIEW

File: search/find-address/README.md [hunk 1]
```diff
@@ -1,27 +1,31 @@
...
+![Image of find address](FindAddress.gif)
+
+## Use case
+
+A user can input a raw address into your app's search bar and zoom to the address location.
 
 ## How to use the sample
 
-For simplicity, the sample comes loaded with a set of addresses. You can select an address to perform online geocoding and show the matching results on the ArcGISMap.
+Select an entry or type in an address in from the drop-down menu at the top of the screen. Suggestions will appear as text is entered. Click a suggestion or hit return to see the address marked with a pin. Tapping on the pin will show the address in a callout.
```
Comment (Rachael-E) [inline search/find-address/README.md]:
```suggestion
Select an entry from the drop-down menu at the top left of the screen to zoom to the address, marked with a pin. 
```

File: search/find-place/README.md [hunk 2]
```diff
@@ -1,34 +1,48 @@
...
+![Image of find place](FindPlace.png)
+
+## Use case
+
+When getting directions or looking for nearby places, users may only know what the place has ("food"), the type of place ("gym"), or the generic place name ("Starbucks"), rather than the specific address. You can get suggestions and locations for these places of interest (POIs) using a natural language query. Additionally, you can filter the results to a specific area.
 
 ## How to use the sample
 
-Choose from the dropdown or input your own place and location to search near. Click the search button to find matching places. A redo search button will appear if you pan the map after a search.
+Choose a type of place in the first field and an area to search within in the second field. Click the 'Search' button to show the results of the query on the map. Click on a result pin to show its name and address. If you pan away from the result area, the "Redo search in this area" button at the bottom of the screen will become active. Click it to query again for the currently viewed area on the map.
```
Comment (Rachael-E) [inline search/find-place/README.md]:
```suggestion
Choose a type of place in the first field and an area to search within in the second field. Click the "Search" button to show the results of the query on the map. Click on a result pin to show its name and address. If you pan away from the result area, the "Redo search in this area" button at the bottom of the screen will become active. Click it to query again for the currently viewed area on the map.
```

File: search/find-place/README.md [hunk 3]
```diff
@@ -1,34 +1,48 @@
...
-3. Create a `LocatorTask` using a URL and set the `GeocodeParameters`.
-4. To reverse geocode near a location, pass the location's position into `GeocodeParameters.setSearchArea(Geometry)` to set the search area.
-5. Limit results to the view's visible area using the `MapView.getVisibleArea()` method.
-6. Show the matching retrieved results from the `LocatorTask.geocodeAsync(String, GeocodeParameters)` via `PictureMarkerSymbol`s with a `Graphic` in a `GraphicsOverlay`.
+1. Create a `LocatorTask` using a URL to a locator service.
+2. Find the location for an address (or city name) to build an envelope to search within:
+    * Create `GeocodeParameters`.
+    * Add return fields to the parameters' `resultAttributeNames` collection. Only add a single "\*" option to return all fields.
+    * Call `locatorTask.geocodeAsync(locationQueryString, geocodeParameters)` to get a list of `GeocodeResult`s.
+    * Use the `displayLocation` from each of the results to build an `Envelope` to view.
```
Comment (Rachael-E) [inline search/find-place/README.md]: can you double check suggestion please: I think this is what this line refers to. 
```suggestion
    * Use `getDisplayLocation` from each of the results to build an `Envelope` to view.
```

File: search/find-place/README.md [hunk 4]
```diff
@@ -1,34 +1,48 @@
...
+3. Get place of interest (POI) suggestions based on a place name query:
+    * Create `SuggestParameters`.
+    * Add "POI" to the parameters' `categories` collection.
```
Comment (Rachael-E) [inline search/find-place/README.md]:
```suggestion
    * Add "POI" to the parameters' categories collection with `getCategories().add("POI")`.
```

File: search/find-place/README.md [hunk 5]
```diff
@@ -1,34 +1,48 @@
...
+    * Call `locatorTask.suggestAsync(placeQueryString, suggestParameters)` to get a list of `SuggestResults`.
```
Comment (Rachael-E) [inline search/find-place/README.md]:
```suggestion
    * Call `locatorTask.suggestAsync(placeQueryString, suggestParameters)` to get a list of `SuggestResult`s.
```

File: search/find-place/README.md [hunk 6]
```diff
@@ -1,34 +1,48 @@
...
+    * The `SuggestResult` will have a `label` to display in the search suggestions list.
```
Comment (Rachael-E) [inline search/find-place/README.md]:
```suggestion
    * The `SuggestResult` will have a label to display in the search suggestions list.
```

File: search/find-place/README.md [hunk 7]
```diff
@@ -1,34 +1,48 @@
...
+4. Use one of the suggestions or a user-written query to find the locations of POIs:
+    * Create `GeocodeParameters`.
+    * Set the parameters' `searchArea` to the envelope.
```
Comment (Rachael-E) [inline search/find-place/README.md]:
```suggestion
    * Set the parameters' search area to the envelope.
```

File: search/reverse-geocode-online/README.md [hunk 8]
```diff
@@ -1,31 +1,30 @@
-# Reverse Geocode Online
+# Reverse geocode online
 
-Find the address of a location.
+Use an online service to find the address for a tapped point.
```
Comment (Rachael-E) [inline search/reverse-geocode-online/README.md]: tapped is very platform specific, surprised this made it through common samples approval
```suggestion
Use an online service to find the address for a point on the map.
```

File: search/offline-geocode/README.md [hunk 9]
```diff
@@ -1,36 +1,29 @@
...
+![Image of offline geocode](OfflineGeocode.png)
+
+## Use case
+
+You can use an address locator file to geocode addresses and locations. For example, you could provide offline geocoding capabilities to field workers repairing critical infrastructure in a disaster when network availability is limited.
 
 ## How to use the sample
 
-For simplicity, the sample comes loaded with a set of addresses. You can select any address to perform geocoding and show it's location on map. You can select the pin and move the mouse to perform reverse geocoding in real-time.
+Select an address from the drop-down list to geocode the address and view the result on the map. Click the location you want to reverse geocode. Click on the pin to highlight it and then click and and drag on the map to get real-time geocoding.
```
Comment (Rachael-E) [inline search/offline-geocode/README.md]:
```suggestion
Select an address from the drop-down list to geocode the address and view the result on the map. Click the location you want to reverse geocode. Click on the pin to highlight it and then move the mouse to perform reverse geocoding in real-time.
```

**Review by Rachael-E — COMMENTED** [review]
@JonLavi oops looks like I missed this one yesterday. a few things to address here, thanks

File: search/reverse-geocode-online/README.md (hunk 8 above)
Comment (JonLavi) [reply search/reverse-geocode-online/README.md]: Good catch!

**Review by Rachael-E — APPROVED** [review]

**Pull request merged**